index.html

```diff
@@ -93,15 +93,15 @@ <h2>Switch panel</h2>
             The HVAC (heating, ventilating, and air-conditioning) applications and lighting applications are usually deployed in homes and buildings, to manage the indoor environment. A switch panel is used to control the HVAC application and lighting application.
           </p>
           <p>
-            <a href="#figure-miniapp-iot-use-case-switch-panel-screen-and-button">Figure 1</a> shows an example of HVAC application switch panel which has a screen and several physical buttons. In this case, by introducing MiniApp for IoT, the switch panel’s screen UI can be designed and developed using Web technologies, to display the HVAC information. For example, the target temperature value inputted by the physical and the measured real-time temperature value can be displayed.
+            <a href="#figure-miniapp-iot-use-case-switch-panel-screen-and-button"></a> shows an example of HVAC application switch panel which has a screen and several physical buttons. In this case, by introducing MiniApp for IoT, the switch panel’s screen UI can be designed and developed using Web technologies, to display the HVAC information. For example, the target temperature value inputted by the physical and the measured real-time temperature value can be displayed.
```

Comment:
```suggestion
            [[[#figure-miniapp-iot-use-case-switch-panel-screen-and-button]]] shows an example of HVAC application switch panel which has a screen and several physical buttons. In this case, by introducing MiniApp for IoT, the switch panel’s screen UI can be designed and developed using Web technologies, to display the HVAC information. For example, the target temperature value inputted by the physical and the measured real-time temperature value can be displayed.
```

We can use this simpler syntax (same for other places).